app/bearing_plotter.rb

```diff
@@ -0,0 +1,13 @@
+module BearingPlotter
+  extend Discordrb::Commands::CommandContainer
+
+  command :bearing, description: "Import of http://edbearingcalc.neocities.org, for your easy of use. X co-ords are the top number, Y is the middle number. (gravity is the bottom number).", useage: "<Your position's X> <Your position's Y> <Your Destination's X> <Your Destination's Y>", min_args: 4, max_args: 4 do |event, x_start, y_start, x_end, y_end|
```

Comment:
`usage` not `useage`. Also, the usage in the description itself is unnecessary, as it's in `usage`.